### DISCUSSION TITLE
Escape content in JSX with clsx

### DISCUSSION THREAD

**Discussion by ofk** [tailwindlabs/tailwindcss#13843] [Help]
The feature of https://tailwindcss.com/docs/content#using-spaces-and-underscores does not work if written in JSX with clsx.

```
<div className="before:content-['Hello\_1']" /> // working
<div className={cx("before:content-['Hello\\_2']")} /> // not working
```

I suspect that the escape sequence in JSX is incorrect, but is there a correct way to escape it?

**Comment by wongjn** [comment]
As per [the documentation](https://tailwindcss.com/docs/adding-custom-styles#handling-whitespace):

> If you’re using something like JSX where the backslash is stripped from the rendered HTML, use [String.raw()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/raw) so the backslash isn’t treated as a JavaScript escape character:
> ```jsx
> <div className={String.raw`before:content-['hello\_world']`}>
>   
> </div>
> ```
